checklist.html

```diff
@@ -64,27 +64,29 @@
tem" id="location0x00017519"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Wariel" target="myframe" class="locationName">Fort Wariel</a><input type="checkbox" class="locationCheck check" id="location0x00017519check"><a href="./map.html?formId=0x00017519&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x000174C5"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Hastrel" target="myframe" class="locationName">Fort Hastrel</a><input type="checkbox" class="locationCheck check" id="location0x000174C5check"><a href="./map.html?formId=0x000174C5&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x000174D4"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Linchal" target="myframe" class="locationName">Fort Linchal</a><input type="checkbox" class="locationCheck check" id="location0x000174D4check"><a href="./map.html?formId=0x000174D4&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x000174FC"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Ontus" target="myframe" class="locationName">Fort Ontus</a><input type="checkbox" class="locationCheck check" id="location0x000174FCcheck"><a href="./map.html?formId=0x000174FC&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x000174D0"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Istirus" target="myframe" class="locationName">Fort Istirus</a><input type="checkbox" class="locationCheck check" id="location0x000174D0check"><a href="./map.html?formId=0x000174D0&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x00017501"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Dirich" target="myframe" class="locationName">Fort Dirich</a><input type="checkbox" class="locationCheck check" id="location0x00017501check"><a href="./map.html?formId=0x00017501&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="location0x000174EC"><img class="itemIcon"  draggable="false" src="images/Icon_Fort.png"><a href="https://en.uesp.net/wiki/Oblivion:Fort_Rayles" target="myframe" class="locationName">Fort Rayles</a><input type="checkbox" class="locationCheck check" id="location0x000174ECcheck"><a href="./map.html?formId=0x000174EC&amp;zoom=0.7" target="myframe" class="itemHelp" title="View on map">🗺️</a></div><div class="location item" id="locat
```

Comment:
oh my lord

Reply:
Yeah 😂 it's all the generated content just splorped into one line. We could manually linebreak it but we'd have to do that every time the content changes so it's not really worth it